hazelcast/src/test/java/com/hazelcast/client/txn/ClientTxnUniSocketTest.java

```diff
@@ -65,6 +65,23 @@ public Member next() {
                 }
                 return members[0];
             }
+
+            @Override
+            public Member nextDataMember() {
+                Member[] members = getDataMembers();
+                if (members == null || members.length == 0) {
+                    return null;
+                }
+                if (members.length == 2) {
```

Comment:
Why do we return the last member in this case?

Reply:
I do not know, this is just a copy-paste from the `next()` method to let the test compile. There is this comment above:
`//try to force second member to connect when exist`